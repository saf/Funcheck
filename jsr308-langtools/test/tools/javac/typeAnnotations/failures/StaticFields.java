/*
 * @test /nodynamiccopyright/
 * @bug 6843077
 * @summary static field access isn't a valid location
 * @author Mahmood Ali
 * @compile/fail/ref=StaticFields.out -XDrawDiagnostics -source 1.7 StaticFields.java
 */
class C {
  int f;
  int a = @A C.f;
}

@interface A { }
