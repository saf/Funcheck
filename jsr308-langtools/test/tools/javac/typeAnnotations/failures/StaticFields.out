StaticFields.java:10:17: compiler.err.illegal.start.of.expr
1 error
